pkg/scaffold/helm/chart.go

```diff
@@ -19,12 +19,17 @@ import (
 	"io/ioutil"
 	"os"
 	"path/filepath"
+	"regexp"
+	"sort"
 	"strings"
 
+	"github.com/markbates/inflect"
```

Comment:
According to that project it's deprecated, we should be using github.com/gobuffalo/flect instead, any reason why you choose this one?

Reply:
The use of the discovery API means we no longer need to guess at the plural resource name, so this is N/A now.